experimental/web_dashboard/lib/src/app.dart

```diff
@@ -2,58 +2,102 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:flutter/material.dart';
 import 'package:provider/provider.dart';
+import 'package:web_dashboard/src/api/api.dart';
```

Comment:
Were these auto-imported by IntelliJ using the wrong schema? It looks like the old imports are path-based, and these new ones are package-based.